library/std/src/os/windows/process.rs

```diff
@@ -199,14 +199,14 @@ pub trait CommandExt: Sealed {
 
     /// Append literal text to the command line without any quoting or escaping.
     ///
-    /// This is useful for passing arguments to applications which doesn't follow
+    /// This is useful for passing arguments to applications that don't follow
     /// the standard C run-time escaping rules, such as `cmd.exe /c`.
     ///
-    /// # Bat files
+    /// # Batch files
```

Comment:
There are many kinds of batch files used by many different job control languages. I believe only the `.bat` files of Windows are under discussion here. This retitling seems to be _less_ correct.

Reply:
I think the proper name is "DOS Batch file" but figured the DOS part is implied when discussing Windows. I have indeed seen "BAT File" though, just not 🦇

Reply:
BAT file seems like the one we might prefer here, yes.

However, I must wonder... do all of these caveats apply to files with the `.cmd` extension also?

Reply:
The documentation so far doesn't seem to mention `.cmd` at all, despite that also being an applicable extension for batch files with basically-the-same command language. I won't pretend to know what the exact caveats and corner cases are, though.

Reply:
> However, I must wonder... do all of these caveats apply to files with the `.cmd` extension also?

Hey, according to wiki .cmd is just another "Batch File" 😄 https://en.wikipedia.org/wiki/Batch_file

Reply:
Indeed!

I had a frisson of annoyance at that, as technically, there are many JCLs etc. etc. and then I realized that while indeed even in 2024 some people do use mainframes with such batch files that are *not* in the MS-DOS command language... everyone understands the meaning here from context, obviously.